admin_manual/installation/examples/lets_encrypt/nginx.rst

```diff
@@ -0,0 +1,183 @@
+Setup NGINX
+===========
+
+The following is an example setup process for NGINX, please adapt it to your exact needs.
+
+NGINX ssl_dhparam
+-----------------
+
+If not already present, add an `ssl_dhparam`_ directive and a new certificate with stronger keys (which improves `forward secrecy`_). 
+The OpenSSL command may take a while to complete, so please be patient. 
+You can place the certificate into any directory you choose.
+
+::
+
+  sudo openssl dhparam -out /etc/nginx/dh4096.pem 4096
+  
+Add the following directive to your common SSL configuration:
+
+.. code-block:: nginx
+
+  ssl_dhparam /etc/nginx/dh4096.pem;
+
+Prepare a server directive for port 443 
+---------------------------------------
+
+It is easiest, if you create a separate file for the following ``ssl_*`` directives. 
+If these directives already exist in this server block, delete them and include the file instead. 
+When the certificate has been created, you can use this file in any SSL server block for which the certificate is valid without 
+reissuing.
+
+::
+
+  cd /etc/nginx/
+  sudo mkdir ssl_rules
+  
+Create a file named ``ssl_mydom.tld`` in the newly created directory. 
+
+.. code-block:: nginx
+
+   # SSL rules for mydom.tld
+   # eases letsencrypt initial cert issuing
+
+   ssl on;
+
+   ssl_certificate         /etc/letsencrypt/live/mydom.tld/fullchain.pem;
+   ssl_certificate_key     /etc/letsencrypt/live/mydom.tld/privkey.pem;
+   ssl_trusted_certificate /etc/letsencrypt/live/mydom.tld/cert.pem;
+
+   ssl_stapling on;
+   ssl_stapling_verify on;
+   ssl_session_timeout 5m;
+
```

Comment:
You have killed the ``/.well-known/acme-challenge`` part for port 80. See my first commit. 
**Mandatory, will not work without...**

Reply:
See above.